### ISSUE TITLE
captureBreadcrumb does not store data

### ISSUE THREAD

**Issue by solussd** [getsentry/react-native-sentry#159]
The React method `captureBreadCrumb` uses a `BreadcrumbBuilder` to build a Breadcrumb data object. BreadcrumbBuilder has a `setData` method, but it is not being used in `captureBreadCrumb`.

**Comment by HazAT** [comment]
Hey, sorry I am not really sure what you mean. 
But if you want to pass additional data to a breadcrumb just do this:
```
Sentry.captureBreadcrumb({
  message: 'Item added to shopping cart',
  category: 'action',
  data: {
     isbn: '978-1617290541',
     cartSize: '3'
  }
});
```

**Comment by solussd** [comment]
@HazAT Sorry, I wasn't clear. I'm referring to the Android React method `captureBreadcrumb`: https://github.com/getsentry/react-native-sentry/blob/master/android/src/main/java/io/sentry/RNSentryModule.java#L164

**Comment by HazAT** [comment]
@solussd Good catch 👍 
Has been fixed `0.14.8`

**Comment by solussd** [comment]
thanks!
